filer/__init__.py

```diff
@@ -1,3 +1,5 @@
 #-*- coding: utf-8 -*-
 # version string following pep-0396 and pep-0386
-__version__ = '0.9pbs.59'  # pragma: nocover
+__version__ = '0.9pbs.102'  # pragma: nocover
```

Comment:
The initial plan was to merge the django-upgrade22 branch sooner into master. I thought it would be a good idea to leave room if we needed to do fixes before the django upgrade goes live.
0.9pbs.60 - 0.9pbs.99: fixes for Django 1.8
>= 0.9pbs.100: Django 2.2 upgrade

Reply:
the problem with this approach is that it doesn't track versions, it tracks commits.
we should have .dev builds with incremental commits in version to track that, because .dev builds are not final builds. look all over pypi, there are no jumps in the versions like this. since we cannot overwrite another commit, i suggest we either add .60.102 at the end, and have .60.0 at deploy, or switch to using the date. this way of versioning is really confusing.